### ISSUE TITLE
KeyError: 'Box 1 chunk0'

### ISSUE THREAD

**Issue by Leopiglet** [facebookresearch/VideoPose3D#81]
Hello darlopavllo, there is a problem that though successfully getting "converted_15j" by Matlab, in next step using "data/prepare_data_humaneva.py" to get ".npz", the two produced files -- "data_2d_humaneva15_gt.npz" and "data_3d_humaneva15.npz" was so small (just 826B and 416B) that perhaps it caused the error when running "python run.py .... ".The detail is as follows:
Loading dataset...
Preparing data...
Loading 2D detections...
Selected actions: ['Walk', 'Jog', 'Box']
Traceback (most recent call last):
File "run.py", line 169, in
cameras_valid, poses_valid, poses_valid_2d = fetch(subjects_test, action_filter)
File "run.py", line 136, in fetch
if parse_3d_poses and 'positions_3d' in dataset[subject][action]:
KeyError: 'Box 1 chunk0'

In short, I guess these two files, "data_2d_humaneva15_gt.npz" and "data_3d_humaneva15.npz" , maybe have some unknown errors, please give me hand, thanks a lot!

**Comment by thang7345** [comment]
Same error here can anyone help!

**Comment by ahmedaskar18** [comment]
you have to run the **ConvertHumanEva.m** file first to get the 3d corrds installed in the **converted_15j**

**Comment by kong-johnny** [comment]
> 

But he/she had already got converted_15j.
